DOCSP-42126 Atlas performance guidance (#204)
mongodb/docs-relational-migrator: source/jobs/creating-jobs.txt

          - Integer. Indicates the number of errors after which Relational 
            Migrator stops the migration job.  
        * - Verify migrated data 
          - Boolean. If true, the sync engine verifies migrated data 
            against the source database. Only supported for snapshot mode.

      When you set the :guilabel:`Mode`, Relational Migrator checks if 

>       the database is configured correctly. If it finds issues, it displays a 

Reviewer: [question] is there any specificity we can add about what a "correct" configuration is?

Author: It's covered in the [Prerequisites](https://deploy-preview-204--docs-relational-migrator.netlify.app/jobs/prerequisites/) sections for the respective databases, but even then, it varies so widely that there's no useful way to summarize it. Solving it's one of the major problems RM in general exists to address, Pre-Migration App Analysis in 1.13 will be another step towards that.

      your migration job. These details include the type, the number of 
      tables and rows affected, and the estimated data size of the 
      migration job. To start your migration job, click :guilabel:`Start`. 
         

>       For Kafka migration jobs, view deployment details by selecting the 
>       job in the :guilabel:`History` pane on the 
>       :guilabel:`Data Migration` screen and hovering
>       over the icon next to the :guilabel:`Create Migration Job`
>       button.

Reviewer: [suggestion] for clarity and active voice 

```suggestion
      To view deployment details for Kafka migration jobs, select the 
      job in the :guilabel:`History` pane on the 
      :guilabel:`Data Migration` screen and hover
      over the icon next to the :guilabel:`Create Migration Job`
      button.
```

      .. important::

>         When running a large or long-running migration job to an Atlas 
>         cluster, the Review Summary includes :guilabel:`Atlas Performance 
>         Suggestions` if the cluster tier creates a bottleneck. Upgrading your 
>         cluster is optional, but reduces migration time when performance is a 
>         priority.

Reviewer: [suggestion] to [start with the conditional ](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#:~:text=Sentences%20that%20describe%20a%20condition%20should%20always%20begin%20with%20the%20conditional%20clause%2C%20so%20that%20the%20user%20can%20skip%20the%20section%20if%20the%20condition%20does%20not%20apply%20to%20them.)

```suggestion
         If the cluster tier creates a bottleneck when you run a large or long-running migration job to an Atlas 
        cluster, the Review Summary includes :guilabel:`Atlas Performance 
        Suggestions`. Upgrading your 
        cluster is optional, but reduces migration time when performance is a 
        priority.
```

Author: It shows the suggestions before running the job, I'll take another pass at the wording.

      tables and rows affected, and the estimated data size of the 
      migration job. To start your migration job, click :guilabel:`Start`. 
         
      To view deployment details for Kafka migration jobs, select the 
      job in the :guilabel:`History` pane on the 
      :guilabel:`Data Migration` screen and hover
      over the icon next to the :guilabel:`Create Migration Job`
      button.

      .. important::

        For large migration jobs to an Atlas cluster, the Review Summary may 
        include :guilabel:`Atlas Performance Suggestions` that recommend 

>         higher cluster tiers. Upgrading is optional, but reduces migration time 
>         when performance is a priority.

Reviewer: [suggestion/question] Does this mean that upgrading reduces migration time only in the case of when performance is a priority or is it that upgrading reduces migration time which may be useful when performance is a priority?

```suggestion
        higher cluster tiers. Upgrading is optional, but may reduce migration time.
```

Author: Changed to "but reduces migration time." with a full stop. I think we're safe stating it as an absolute. 

The UI text for it in Relational Migrator is, "MongoDB performance testing indicates that your database is using a lower cluster tier. If migration performance is important, please consider upgrading your Atlas instance to a faster tier." so it covers the rest.
